src/aliases.jl

```diff
@@ -1,3 +1,26 @@
+import AbstractAlgebra: LaurentMPolyRing  # FIXME: remove these imports once AA exports these
+import AbstractAlgebra: LaurentMPolyRingElem
+import AbstractAlgebra: LaurentPolyRing
+import AbstractAlgebra: LaurentPolyRingElem
+include(joinpath(pathof(AbstractAlgebra), "..", "Aliases.jl"))
+
+import Nemo: is_cyclo_type
+import Nemo: is_embedded
+import Nemo: is_maxreal_type
+import Nemo: ZZModRing  # FIXME: remove if/once Nemo exports this
+import Nemo: zzModRing  # FIXME: remove if/once Nemo exports this
+import Nemo: FpField  # FIXME: remove if/once Nemo exports this
+import Nemo: fpField  # FIXME: remove if/once Nemo exports this
+include(joinpath(pathof(Nemo), "..", "Aliases.jl"))
+
+#import Hecke: quadratic_genera
+#import Hecke: has_algebra
+#import Hecke: has_embedding
+#import Hecke: has_matrix_action
+#import Hecke: has_root
+#import Hecke: ...
+#include(joinpath(pathof(Hecke), "..", "Aliases.jl"))
+
```

Comment:
I guess we could remove this... I initially thought it would be good to have Hecke compat, too, but there are a *ton* of things in Hecke for which we have deprecated aliases but which never were exported... So it is a lot of work to adjust this. I can do it, but only if someone (@thofma, @fieker, ...) tells me it is important.
```suggestion
```